src/person.cpp

```diff
@@ -1617,6 +1664,73 @@ AnimateResult Guest::VisitRideOnAnimate(RideInstance *ri, TileEdge exit_edge)
 	return OAR_CONTINUE;
 }
 
+/** Pixel positions of guests sitting on a bench. */
+static XYZPoint16 _bench_pix_pos[4 /* TileEdge */][2 /* Left = 0, Right = 1 */] = {
+	{XYZPoint16(  0, 160, 0), XYZPoint16(  0,  96, 0)},
+	{XYZPoint16(160, 255, 0), XYZPoint16( 96, 255, 0)},
+	{XYZPoint16(255,  96, 0), XYZPoint16(255, 160, 0)},
+	{XYZPoint16( 96,   0, 0), XYZPoint16(160,   0, 0)},
+};
```

Comment:
Is this universal, or just happens to be fit for your benches?
If the latter, please consider moving it to the rcd files at some point.

Reply:
This is pretty much the only configuration that allows two guests to sit on the same tile edge without overlapping with each other or any neighbouring people, edges or voxels. So I'd consider it universal.